cmd/kubelet/app/server.go

```diff
@@ -339,6 +227,88 @@ func validateConfig(s *options.KubeletServer) error {
 	return nil
 }
 
+// if the node config controller was used, it will be returned as the first return value
+func initNodeConfig(s *options.KubeletServer, client clientset.Interface, nodeName string, standaloneMode bool) (*nodeconfig.NodeConfigController, error) {
```

Comment:
Ah nvm I see it. Temporarily forgot that the first is necessary for the initial set of gates.